frontend/packages/console-dynamic-plugin-sdk/README.md

```diff
@@ -32,6 +32,17 @@ dynamic-demo-plugin/
 | `@openshift-console/dynamic-plugin-sdk-webpack` | Provides webpack plugin `ConsoleRemotePlugin` used to build all dynamic plugin assets. |
 | `@openshift-console/dynamic-plugin-sdk-internal` | Internal package exposing additional code. |
 
+## OpenShift Console Versions vs SDK Versions
+
+Not all NPM packages are fully compatible with all versions of the Console. This table will help align compatible versions of the SDK Packages to versions of the OpenShift Console.
+
+| Console Version | SDK Package | Last Package Version |
+| --- | --- | --- |
+| 4.11.x | `@openshift-console/dynamic-plugin-sdk` | Latest |
+|  | `@openshift-console/dynamic-plugin-sdk-webpack` | Latest |
+| 4.10.x | `@openshift-console/dynamic-plugin-sdk` | 0.0.3 |
+|  | `@openshift-console/dynamic-plugin-sdk-webpack` | 0.0.4 |
```

Comment:
Used Code Freeze dates to match against versions that were released at that time.